Cargo.toml

```diff
@@ -12,6 +12,9 @@ readme = "README.md"
 
 build = "build.rs"
 
+[profile.release]
+debug = true
```

Comment:
This'll possibly bloat code size a bit, so if this is just an artifact from your testing it should probably be removed.

Reply:
It is an artifact of testing - will remove!